doc/logging.md

```diff
@@ -0,0 +1,32 @@
+# Compile time log levels
+
+At compile time the maximum log level can be specified. This allows distros to
+disable the overhead code and reduce overall code size.
+
+# Runtime log level
+
+At runtime, the log level is determined by an environment variable. The default
+log level is WARNING. The level can be changed by setting the TSS2_LOG
```

Comment:
It looks like the default is INFO, set in log/log.h

Reply:
Done